Using npm-link while developing plugin fails

**/test-server/index.js**
```
const fastify = require('fastify')()
fastify.register(require('fastify-example'), (err) => {
  if (err) throw err
})
```

**/fastify-example/plugin.js**
```
const fp = require('fastify-plugin')
function plugin (fastify, opts, next) {
  // do something cool
  next()
}
module.exports = fp(plugin, '>=0.15.0')
```

-----

```
/fastify-example/> npm link
/fastify-example/> cd ..
/> cd test-server
/test-server/> npm link fastify-example
/test-server/> node index.js
Error: Cannot find module 'fastify/package.json'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at plugin (/fastify-example/node_modules/fastify-plugin/index.js:15:59)
    at Object.<anonymous> (/fastify-example/plugin.js:6:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
```

--- a/index.js
+++ b/index.js
@@ -1,6 +1,7 @@
 'use strict'
 
 const semver = require('semver')
+const console = require('console')
 
 function plugin (fn, version) {
   if (typeof fn !== 'function') {
@@ -12,8 +13,14 @@ function plugin (fn, version) {
       throw new TypeError(`fastify-plugin expects a version string as second parameter, instead got '${typeof version}'`)
     }
 
-    const fastifyVersion = require('fastify/package.json').version
-    if (!semver.satisfies(fastifyVersion, version)) {
+    var fastifyVersion
+    try {
+      fastifyVersion = require('fastify/package.json').version
+    } catch (_) {
+      console.info('fastify not found, proceeding anyway')
+    }
+
+    if (fastifyVersion && !semver.satisfies(fastifyVersion, version)) {
       throw new Error(`fastify-plugin - expected '${version}' fastify version, '${fastifyVersion}' is installed`)
     }
   }

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/fastify/fastify-plugin#readme",
   "devDependencies": {
     "fastify": "^0.18.0",
+    "proxyquire": "^1.8.0",
     "standard": "^10.0.0",
     "tap": "^10.3.0"
   },

--- a/test.js
+++ b/test.js
@@ -1,6 +1,7 @@
 'use strict'
 
 const t = require('tap')
+const proxyquire = require('proxyquire')
 const test = t.test
 const fp = require('./')
 
@@ -71,4 +72,23 @@ test('should throw if the version number is not a string', t => {
   } catch (e) {
     t.is(e.message, 'fastify-plugin expects a version string as second parameter, instead got \'number\'')
   }
+})
+
+test('should not throw if fastify is not found', t => {
+  t.plan(1)
+
+  const fp = proxyquire('./index.js', {
+    'fastify/package.json': null,
+    console: {
+      info: function (msg) {
+        t.is(msg, 'fastify not found, proceeding anyway')
+      }
+    }
+  })
+
+  function plugin (fastify, opts, next) {
+    next()
+  }
+
+  fp(plugin, '>= 0')
 })

